api/app/src/Repository/PreRegistrationRepository.php

```diff
@@ -60,4 +60,48 @@ public function findByCaseNumber(string $caseNumber)
             ->getQuery()
             ->getResult();
     }
+
+    public function getNewClientsForExistingDeputiesArray(): array
+    {
+        $conn = $this->getEntityManager()->getConnection();
```

Comment:
I think we would need to disable the softdeletable filter here before running the query.
The reason being is that when we probably should include the discharged clients in the check so that if they appear in the CSV for this particular deputy, they aren't recreated again.

Reply:
Given that we're going for strings as that's whats on sirius for deputy uid I'd do the cast the other way round and cast the bigint (that we're ditching eventually) as a string (or varchar or whatever it's called in database language) instead

Reply:
Is it just a case of removing the softdeletable filter Gugs on line 103? I think the SQL should bring back all the discharged ones fine but then the filter gets rid of them right?

Reply:
I believe you have to disable it before the query, run the query (it will now ignore the filter), and then enable it again afterwards so it doesn't affect any other queries afterwards.

Thinking about it, it's not the greatest implementation because if you've got a query that is going to run for a long time then the filter is disabled for a long time too. Which means there's a higher chance some other query in the app is going to be running and that might be expecting the filter to be enabled.